rules-tests/Php81/Rector/Property/ReadOnlyPropertyRector/Fixture/skip_api-platform-api-resource.php.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace Rector\Tests\Php81\Rector\Property\ReadOnlyPropertyRector\Fixture;
+
+use ApiPlatform\Metadata\ApiResource;
+
+#[ApiResource]
```

Comment:
is `@ApiResource` annotation also exists?

Reply:
In actual versions - no.

Reply:
in v2.5, annotation is allowed, so I think annotation can be registered as well

https://api-platform.com/docs/v2.5/core/serialization/

Reply:
ok, added.